app/_plugins/drops/plugins/examples/konnect.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module Drops
+    module Plugins
+      module Examples
+        class Konnect < Base
+          URLS = {
+            'consumer' => 'https://global.api.konghq.com/v2/{RUNTIME-GROUP-ID}/core-entities/consumers/{CONSUMER_NAME|ID}/plugins',
```

Comment:
The global endpoint won’t work here. It needs to be us.api or eu.api

Reply:
Is that just for consumers, or consumer groups as well?

Reply:
For all APIs that aren't kauth related